app/helpers/common_parameters_helper.rb

```diff
@@ -14,7 +14,8 @@ def parameter_value_field(value)
     content_tag :div, :class => "form-group condensed" do
       text_area_tag("value_#{value[:safe_value]}", value[:safe_value], :rows => (value[:safe_value].to_s.lines.count || 1 rescue 1),
                     :class => "col-md-5", :disabled => true, :'data-hidden-value' => Parameter.hidden_value) +
-      content_tag(:span, :class => "help-block") { popover(_("Additional info"), _("<b>Source:</b> %{type} %{name}") % {:type => _(value[:source].to_s), :name => source_name})}
+      content_tag(:span, :class => "col-md-2") { textarea_expander } +
+      content_tag(:span, :class => "help-inline col-md-3") { popover(_("Additional info"), _("<b>Source:</b> %{type} %{name}") % {:type => _(value[:source].to_s), :name => source_name})}
```

Comment:
Nitpick, here and in other files, the second line should be indented so it's clear that this is one expression